config/settings.defaults.js

```diff
@@ -22,6 +22,10 @@ module.exports = {
 
   healthCheckUserId: '53c64d2fd68c8d000010bb5f',
 
+  ignoredMisspellings: process.env.IGNORED_MISSPELLINGS
+    ? process.env.IGNORED_MISSPELLINGS.split(',')
+    : ['ShareLaTeX', 'sharelatex', 'LaTeX', 'http', 'https', 'www'],
```

Comment:
shall we add `overleaf` in as well?

Reply:
:+1: They are not marked as misspelled in English, but are marked as such in e.g. French -- done in f47cac4cff935e1b29630656473b25bc1ff1df95